src/Propagation/Setter.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I don't think `declare(strict_types=1)` do anything in files that only contain an interface declaration. Will our linter complain?

Reply:
I haven't seen any warnings and all previous interfaces seem to have `declare(strict_types=1)` as well.

Reply:
Can you try removing it and running the linter? I don't think it does anything lol

Reply:
I removed it and ran `make style` and it re-applied `declare(strict_types=1)` to the interfaces